### PR TITLE
Add try-except statement

### LINKED ISSUE #6: add try-except statement
https://github.com/prabhakar267/wikipedia-frequency-lookup/blob/master/main.py#L87

### PR DESCRIPTION
**Pull request by lmzyk**
fixes #6

### CODE REVIEW

**Comment by prabhakar267** [comment]
@lmzyk Why did you remove the screenshots?
They are being used in README. 
The code patch looks fine :+1:

File: main.py [hunk 1]
```diff
@@ -84,7 +84,10 @@ def remove_stop_words(frequency_list):
 	search_mode = False
 
 url = wikipedia_api_link + string_query
-response = requests.get(url)
+try:
+    response = requests.get(url)
+except Exception, e:
```
Comment (prabhakar267) [inline main.py]: Even if there was an exception, the rest of code is getting executed, raising different error

Maybe you can shift the whole code in try statement or find some other work around

Comment (prabhakar267) [inline main.py]: Even if there was an exception, the rest of code is getting executed, raising different error

Maybe you can shift the whole code in try statement or find some other work around

**Comment by lmzyk** [comment]
I don't understand about the screenshot. Probably an error when i clone the project...

**Comment by prabhakar267** [comment]
Also, the name of the files (screenshots) is not same in the last commit, so they are not visible in the README.

**Comment by lmzyk** [comment]
I modify the file name path in the README. Windows change ":" to "-"

File: main.py (hunk 1 above)
Comment (lmzyk) [inline main.py]: I can shift the wole code in try statement. If you want an other way to verify if there was an exception, i'll work on it when i have the time! :)

**Comment by prabhakar267** [comment]
@lmzyk rebased and merged your changes :+1:

**Pull request closed without merging**